Apartments_io/DataAccess/Context/DataBaseContext.cs

```diff
@@ -7,7 +7,17 @@ namespace DataAccess.Context
 {
     public class DataBaseContext : DbContext
     {
+        // FIELDS
+
+        // CONSTRUCTORS
+        static DataBaseContext()
+        {
+            Instance = new DataBaseContext();
```

Comment:
Where is private constructor of DataBaseContext?

Reply:
There is no. I think it better to have public default constructor, in case you want to create new DataBaseContext